ql/src/java/org/apache/hadoop/hive/ql/exec/repl/bootstrap/load/table/LoadPartitions.java

```diff
@@ -243,29 +246,34 @@ private void addPartition(boolean hasMorePartitions, AlterTableAddPartitionDesc
               : LoadFileType.OVERWRITE_EXISTING);
       stagingDir = PathUtils.getExternalTmpPath(replicaWarehousePartitionLocation, context.pathInfo);
     }
-
-    Task<?> copyTask = ReplCopyTask.getLoadCopyTask(
-        event.replicationSpec(),
-        new Path(event.dataPath() + Path.SEPARATOR + getPartitionName(sourceWarehousePartitionLocation)),
-        stagingDir,
-        context.hiveConf, false, false
-    );
-
+    Path partDataSrc = new Path(event.dataPath() + File.separator + getPartitionName(sourceWarehousePartitionLocation));
```

Comment:
Can create a path directly from the constructor of Path instead of using File Separator

Reply:
Normally yes, but in this case it will not as getPartitionName(sourceWarehousePartitionLocation) returns value with / attached. Path will take that itself as final path.